src/drivers/barometer/mpl3115a2/mpl3115a2.cpp

```diff
@@ -501,19 +477,16 @@ MPL3115A2::cycle()
 			 * to wait 2.8 ms after issuing the sensor reset command
 			 * according to the MPL3115A2 datasheet
 			 */
-			start_cycle(USEC2TICK(2800));
+			start_cycle((2800));
```

Comment:
There is a pair of double parentheses here.

Reply:
Fixed.